internal/types/backup.go

```diff
@@ -42,22 +63,27 @@ type OperationState string
 type Operation interface {
 	GetId() ObjectID
 	SetId(id ObjectID)
+	GetContainerID() string
```

Comment:
Do we really need `GetContainerId()` in `Operation` interface?

I believe that the best way as of now is to write getters and setters for fields that are used in processor.

And maybe it's easier to just reference `TakeBackupOperation` fields if it is downcasted from `Operation`. This reduces boilerplate.